tests/core/nogo/custom/custom_test.go

```diff
@@ -427,6 +510,12 @@ func Test(t *testing.T) {
 				// note the cross platform regex :)
 				`.*[\\/]cgo[\\/]examplepkg[\\/]pure_src_with_err_calling_native.go:.*function must not be named Foo \(foofuncname\)`,
 			},
+		}, {
+			desc:        "config_flags_triggering_error",
+			target:      "//:has_errors",
+			wantSuccess: false,
+			config:      "flagger_config.json",
+			includes:    []string{"all switches were set -> fail"},
```

Comment:
There is some bad copy paste here. Could you please double check?

Reply:
Sorry what's the bad copy paste here? Is this what's causing the tests to fail on Buildkite?

Reply:
I think your target name is wrong, also wantSuccess value should be changed at least?

Reply:
I think I intended on using these values. At any rate, I've separated out these tests into a `flags_test` package to avoid polluting the more intricate tests here.